### ISSUE TITLE
version 1.5.0a :: cmake -DIQTREE_FLAGS="moi" generate iqtree not iqtree-mpi

### ISSUE THREAD

**Issue by EricDeveaud** [Cibiv/IQ-TREE#12]
Hello, 

following documentation http://www.iqtree.org/doc/Compilation-Guide/#compiling-mpi-version

I issued the following 
```
OMPI_CC=gcc OMPI_CXX=g++ cmake -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx /local/gensoft2/src/IQ-TREE/IQ-TREE-1.5.0a
```

generates following binary: iqtree 
not iqtree-mpi as expected.

best regards

Eric

**Comment by bqminh** [comment]
Hi Eric, this MPI version is currently only available as a separate version, which can be downloaded at http://www.iqtree.org/#variant or from https://github.com/Cibiv/IQ-TREE/releases/tag/v1.4.4-mpi
I updated the compilation guide accordingly. Thanks for pointing out this.
Note that this MPI version will be merged soon to the main version. Will let you know when that happens.
Minh

**Comment by EricDeveaud** [comment]
ok thanks 
we already provide 1.4.4 mpi version to our users, will wait for the next official mpi release.
regards
Eric

**Comment by bqminh** [comment]
Dear Eric, the MPI version is now officially released in version 1.5.1 with more substantial features. Please get it now from the iqtree website.
Minh
